src/screens/Profile/Header/ProfileHeaderLabeler.tsx

```diff
@@ -328,8 +326,8 @@ function CantSubscribePrompt({
       <Prompt.TitleText>Unable to subscribe</Prompt.TitleText>
       <Prompt.DescriptionText>
         <Trans>
-          We're sorry! You can only subscribe to twenty labelers, and you've
-          reached your limit of twenty.
+          We're sorry! You can only subscribe to {MAX_LABELERS} labelers, and
+          you've reached your limit of {MAX_LABELERS}.
```

Comment:
Is this going to jam up the translators?

Reply:
Yeah it might. I linked `MAX_LABELERS` in a comment instead, just wanted to keep a relationship there to avoid forgetting to update one or the other.